archunit-integration-test/src/test/java/com/tngtech/archunit/testutils/CyclicErrorMatcher.java

```diff
@@ -36,9 +36,10 @@ private String detailText() {
 
     private List<String> detailLines() {
         List<String> result = new ArrayList<>();
+        int count = 0;
         for (Map.Entry<String, Collection<ExpectedRelation>> detail : details.asMap().entrySet()) {
-            result.add("Dependencies of " + detail.getKey());
-            result.addAll(transform(detail.getValue(), toStringFunction()));
+            result.add("        " + ++count + ". Dependencies of " + detail.getKey());
```

Comment:
That's a little much `+` :joy: 
But seriously, I think it might be better to actually ignore the white space or the numbering of the dependencies within the integration test. E.g. by using `builder.matchesLine(..)` instead of `builder.containsLine(..)` and then just looking for `".*Dependencies of " + quote(details.getKey) + ".*"` or similar. I don't think it's necessary for the integration test to break if the indent changes from 4 to 2 for example. Or if we decide to not number the dependencies anymore. Important part is that all the details are contained.

Reply:
Glad to hear you say it. :) I think `matchesLine` will be more flexible.